packages/nodes-base/test/nodes/Postgres/Postgres.node.functions.test.js

```diff
@@ -0,0 +1,104 @@
+const PostgresFun = require('../../../nodes/Postgres/Postgres.node.functions')
+const pgPromise = require('pg-promise');
+
+describe('pgUpdate', () => {
```

Comment:
love that we are adding tests, however are just happy paths covered here. Don't know if adding more tests is out of scope of this PR

Reply:
I believe the tests cover the extend of this PR as we're now adding functionality to existing node and this is being covered.

Good that you mentioned this. I added tests to the insert operation that I incremented. It was missing.

Regarding non-happy path, actually the functions don't have much to fail. Only issues that might arise are query issues which would throw regular Errors and would be catched by the node and displayed to our user.